inc/class-core-sitemaps-renderer.php

```diff
@@ -31,10 +31,14 @@ class Core_Sitemaps_Renderer {
 	 * Core_Sitemaps_Renderer constructor.
 	 */
 	public function __construct() {
-		$stylesheet_url         = $this->get_sitemap_stylesheet_url();
+		$stylesheet_url = $this->get_sitemap_stylesheet_url();
```

Comment:
Per https://github.com/GoogleChromeLabs/wp-sitemaps/pull/155#discussion_r408077644, don't introduce new filters, just use the return value from the existing `core_sitemaps_stylesheet_url` and `core_sitemaps_stylesheet_index_url` filters to decide whether to output the `xml-stylesheet` PI.